app/javascript/components/components/Header.js

```diff
@@ -37,7 +38,9 @@ export default class Header extends Component {
     return (
       <div>
         <Navbar color="light" light expand="md">
-          <NavbarBrand href="/">Money Cat</NavbarBrand>
+          <NavbarBrand href="/">
```

Comment:
If this link is providing internal React navigation, it should likely be a `NavLink` from react-router-dom